packages/web-components/src/components/search/search.ts

```diff
@@ -119,6 +179,22 @@ class CDSSearch extends HostListenerMixin(FocusMixin(FormMixin(LitElement))) {
     }
   }
 
+  /**
+   * Move focus back to the magnifier element.
+   * Adds tabindex="-1" if it is not focusable yet.
+   */
+  private _focusMagnifier() {
+    const magnifier = this.shadowRoot?.querySelector(
+      `.${prefix}--search-magnifier`
+    ) as HTMLElement | null;
```

Comment:
```suggestion
    const magnifier = this.shadowRoot?.querySelector<HTMLElement>(
      `.${prefix}--search-magnifier`
    );
```

`<HTMLElement>` can be used here and you get type checking for free (this will make the type `HTMLElement | null`).